test/e2e/network/network_policy.go

```diff
@@ -491,6 +493,49 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			})
 		})
 
+		ginkgo.It("should allow ingress access from namespace on one named port [Feature:NetworkPolicy]", func() {
+			nsBName := f.BaseName + "-b"
+			nsB, err := f.CreateNamespace(nsBName, map[string]string{
+				"ns-name": nsBName,
+			})
+			framework.ExpectNoError(err, "Error creating namespace %v: %v", nsBName, err)
+
+			const allowedPort = 80
+			policy := &networkingv1.NetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "allow-client-a-via-named-port-ingress-rule",
```

Comment:
Seems odd the name describes that client-a should be allowed but on line 535 it is testing that client-a should not connect. I guess the policy name is the incorrect part here.

Reply:
@lzang right, so it should be `allow-client-b-via-named-port-ingress-rule`